sklearn/preprocessing/_encoders.py

```diff
@@ -945,3 +945,34 @@ def inverse_transform(self, X):
                 X_tr[mask, idx] = None
 
         return X_tr
+
+    def get_feature_names(self, input_features=None):
+        """
+        Return feature names for output features.
+
+        Parameters
+        ----------
+        input_features : list of str of shape (n_features,)
+            String names for input features if available. By default,
+            "x0", "x1", ... "xn" is used.
+
+        Returns
+        -------
+        output_feature_names : ndarray of shape (n_output_features,)
+            Array of feature names.
+
+        .. versionadded:: 1.0.dev0
```

Comment:
We anticipate that 1.0 will be the first public release (with a non-dev version number) to receive this feature:

```suggestion
        .. versionadded:: 1.0
```